### PR TITLE
Do not remove "class foo" comments.

### PR DESCRIPTION
**Pull request by carusogabriel**
Discovered in https://github.com/doctrine/mongodb-odm/pull/2014#discussion_r277157744

### CODE REVIEW

**Comment by greg0ire** [comment]
@Ocramius the issue at hand is "Initializes this instance with the specified document manager, unit of work and\nclass metadata." being unduly truncated, "Initializes this instance with the specified document manager, unit of work and\n@see foo" would raise some eyebrows :P . IMO the right fix would be to remove it iff it the line before is `/**`, but that's probably not possible, the best thing after that probably being @carusogabriel 's solution.

File: tests/input/example-class.php [hunk 1]
```diff
@@ -10,7 +10,8 @@
 use Doctrine\Sniffs\Spacing\ControlStructureSniff;
 
 /**
- * Description
+ * Description of the
+ * class example.
```
Comment (greg0ire) [inline tests/input/example-class.php]: The build fails, you are probably supposed to add this in the output class too.

**Comment by malarzm** [comment]
> Issue is easily worked around by splitting the sentence differently.

Come on, this is a hack for a broken rule

**Comment by ostrolucky** [comment]
It's not broken, just imperfect. And don't make "hack" word even more meaningless than it is. Hitting enter at different part of sentence is hardly hacking. Regex could be improved by taking preceding content into consideration. Just making it case sensitive is lazy solution impacting false negatives too much for my liking.

**Comment by ostrolucky** [comment]
> Regex could be improved by taking preceding content into consideration

**Pull request closed without merging**